packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoDataCell.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode } from 'react';
+
+import { EuiToolTip } from '@elastic/eui';
+
+interface WfoDataCellProps {
+    showTooltip: boolean;
+    content: ReactNode;
```

Comment:
If i understand this correct this is the content inside the tooltip ietself and not the actual data in the cell?
Therefore I would suggeste to rename this into `tooltipContent`